lib/apiservers/portlayer/swagger.json

```diff
@@ -1601,16 +1601,10 @@
 						}
 					},
 					"404": {
```

Comment:
With the schema, we were getting this error from swagger while returning, for example, [this error](https://github.com/vmware/vic/blob/master/lib/apiservers/portlayer/restapi/handlers/containers_handlers.go#L343) from the portlayer:
```
Server error from portlayer: Unknown error from the interaction port layer: &{0 } (*models.Error) is not supported by the ByteStreamConsumer, can be resolved by supporting Writer/BinaryUnmarshaler interface
```

This was causing the [type switch on the error](https://github.com/vmware/vic/blob/master/lib/apiservers/engine/backends/container_proxy.go/#L439) to hit `default`.